cmd/pop/cli/off.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"github.com/myelnet/pop/node"
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var offCmd = &ffcli.Command{
+	Name:       "off",
+	ShortUsage: "off",
+	ShortHelp:  "Gracefully shutdown the Pop daemon",
```

Comment:
@tchardin shouldn' it be "Gracefully shuts down..." as well ?

Reply:
@gallexis 👍